src/app/payment/services/auth-checkout.service.ts

```diff
@@ -0,0 +1,73 @@
+import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
+import { RegisterFunction } from 'app/auth/components/SignUp/useSignUp';
+import { doLogin, getNewToken } from 'app/auth/services/auth.service';
+import localStorageService from 'app/core/services/local-storage.service';
+import { decryptPrivateKey } from 'app/crypto/services/keys.service';
+import { AppDispatch } from 'app/store';
+import { planThunks } from 'app/store/slices/plan';
+import { productsThunks } from 'app/store/slices/products';
+import { referralsThunks } from 'app/store/slices/referrals';
+import { initializeUserThunk, userActions, userThunks } from 'app/store/slices/user';
+
+const signUp = async (
```

Comment:
Because I only need the signup and the dispatches. But there are redirects and extra logic we don't need for the integrated checkout. 

See it in [SignUp](https://github.com/internxt/drive-web/blob/master/src/app/auth/components/SignUp/SignUp.tsx#L138) and [Login](https://github.com/internxt/drive-web/blob/master/src/app/auth/components/LogIn/LogIn.tsx#L129).